ebr_trackerbot/storage/db.py

```diff
@@ -0,0 +1,135 @@
+"""
+Slack Bot Generic DB Storage
+"""
+
+import logging
+import pendulum
+import sys
+
+
+def create_table(get_connection):
+    """
+    Create table if it does not exist
+    """
+    conn = get_connection()
+    try:
+        cursor = conn.cursor()
+        cursor.execute(
+            "CREATE TABLE IF NOT EXISTS tracks "
+            + "(id integer PRIMARY KEY, user text NOT NULL, test text NOT NULL, "
+            + " expiry timestamp NOT NULL, channel_id text NOT NULL, thread_ts text NOT NULL);"
+        )
+        conn.commit()
+    except:
+        logging.error(str(sys.exc_info()[1]))
```

Comment:
I don't think that having the program continue if this call fails is really a use-case.
I would instead propose changing this section to a `with` statement, if both libraries will support it with the same syntax (I'm not sure that the SQLite one will).
https://github.com/mkleehammer/pyodbc/wiki/Objects#exception-handling

Reply:
ok. I rewrote it